packages/keyvAdapter/README.md

```diff
@@ -35,3 +35,17 @@ new ApolloServer({
   cache: new KeyvAdapter(new Keyv("redis://...")),
 });
 ```
+
+## Options
+
+### disableBatchReads <boolean>
+
+By default, `KeyvAdapter` will use [`DataLoader`'s batching functionality](https://github.com/graphql/dataloader#batching) in order to request a list of keys when possible. Support for this depends on the `Keyv` implementation that you're using, specifically its `store` must implement a `getMany` function.
```

Comment:
This isn't quite true.  This works no matter what: if `store` has no `getMany` then `Keyv` will do the parallel-join thing.  What's bad is if your `store` *does* have a `getMany` but it *doesn't work*, which is the case for `@keyv/redis` + `Redis.Cluster`.